java/arcs/core/host/ArcHostContextParticle.kt

```diff
@@ -141,7 +144,10 @@ class ArcHostContextParticle(
 
                 ParticleContext(
                     particle,
-                    Plan.Particle(particleEntity.particleName, particleEntity.location, handlesMap)
+                    Plan.Particle(particleEntity.particleName, particleEntity.location, handlesMap),
+                    scheduler,
```

Comment:
Is it possible to make this a required parameter now, and remove the null-handling related behavior in ParticleContext?

Reply:
Done